packages/tabs/README.md

```diff
@@ -0,0 +1,171 @@
+# @undercoatcss/tabs
+
+## Installation
+```sh
+npm install @lovata/tabs
+```
+
+## Initialize Tabs
+```javascript
+import Tabs from '@lovata/tabs';
```

Comment:
The same issue lovata instead of undercoat

Reply:
Yep, copy/paste mistake